### ISSUE TITLE
[Fix checkJs errors]: Cannot find module '@octoherd/cli/run' or its corresponding type declarations

### ISSUE THREAD

**Issue by oscard0m** [octoherd/script-setup-renovate#4]
Cannot find module '@octoherd/cli/run' or its corresponding type declarations.

https://github.com/octoherd/script-setup-renovate/blob/7ddff65634469652e31ba9c926e57323e447a4dc/cli.js#L4

---

Should we improve types in `@octoherd/cli` ?

**Comment by gr2m** [comment]
> Should we improve types in `@octoherd/cli` ?

Yes, however I'm not sure how to add typescript definitions for the `/run` export. The export itself is defined here

https://github.com/octoherd/cli/blob/309a72c7141115805278cfd99a0b4ee7d4b4c092/package.json#L11

Do you know how to add declarations for `@octoherd/cli/run`?
